src/Module.Server/Modes/Dtv/CrpgDtvServer.cs

```diff
@@ -1,30 +1,31 @@
-﻿using Crpg.Module.Api.Models;
-using Crpg.Module.Common;
-using Crpg.Module.Common.Network;
+﻿using System.Xml.Linq;
 using Crpg.Module.Rewards;
+using NetworkMessages.FromClient;
 using TaleWorlds.Core;
 using TaleWorlds.Library;
+using TaleWorlds.ModuleManager;
 using TaleWorlds.MountAndBlade;
 using TaleWorlds.MountAndBlade.MissionRepresentatives;
 using TaleWorlds.MountAndBlade.Network.Messages;
 using TaleWorlds.ObjectSystem;
 
-namespace Crpg.Module.Modes.DTV;
+namespace Crpg.Module.Modes.Dtv;
 
-internal class CrpgDTVServer : MissionMultiplayerGameModeBase
+internal class CrpgDtvServer : MissionMultiplayerGameModeBase
 {
-    private const int TotalRounds = 8;
-    private const int TotalWaves = 3;
+
     private const int BotRespawnTime = 3;
     private const int NewRoundRespawnTime = 20;
 
-    private readonly CrpgDTVClient _dtvClient;
+    private readonly int totalRounds;
+    private readonly int totalWaves;
```

Comment:
Weird that you're not getting a warning here but private fields should start with an underscore